### PR TITLE
Merge/add seabed mask

### PR DESCRIPTION
**Pull request by ruxandra-valcu**

### CODE REVIEW

**Review by mihaiboldeanu — APPROVED** [review]
Looks good! if all tests pass it can be added.

**Review by mihaiboldeanu — CHANGES_REQUESTED** [review]
in [conftest.py](https://github.com/OceanStreamIO/echopype/blob/merge/add_seabed_mask/echopype/tests/conftest.py#L101) there is a duplicate definition of fixture :
raw_dataset_jr179(setup_test_data_jr179):

In tests for [seabed](https://github.com/OceanStreamIO/echopype/blob/5ecea2928b852289d7b48183d76b3a2a9fe818ce/echopype/tests/mask/test_mask_seabed.py#L4) and shoals mask there is an import of fixtures from conftest.py they are not needed to be used by pytest,

File: echopype/mask/seabed.py [hunk 1]
```diff
@@ -0,0 +1,565 @@
...
+def _get_seabed_mask_blackwell_mod(
+    Sv,
+    r,
+    theta=None,
+    phi=None,
+    r0=10,
+    r1=1000,
+    tSv=-75,
+    ttheta=702,  # on raw EK60 data rather than physical angle
+    tphi=282,  # on raw EK60 data rather than physical angle
```
Comment (simedroniraluca) [inline echopype/mask/seabed.py:243]: In this method, do you use the raw angles or the calibrated ones ?

**Review by simedroniraluca — APPROVED** [review]

File: echopype/mask/seabed.py (hunk 1 above)
Comment (ruxandra-valcu) [reply echopype/mask/seabed.py:243]: The threshhold itself is used on the converted-back-to-what-the-raw-EK60-value-would-be, as discussed in our call

File: echopype/mask/api.py [hunk 2]
```diff
@@ -572,3 +580,125 @@ def get_shoal_mask(
...
+
+
+def get_seabed_mask(
+    source_Sv: Union[xr.Dataset, str, pathlib.Path],
+    desired_channel: str,
+    mask_type: str = "ariza",
+    **kwargs,
+) -> xr.DataArray:
+    """
+    Create a mask based on the identified signal attenuations of Sv values at 38KHz.
```
[the remark this answers is not in the document]
Comment (ruxandra-valcu) [reply echopype/mask/api.py:592]: Other frequencies, I'll edit

**Pull request merged**